apps/web/src/pages/TalentNominations/NominationGroupEvaluationDialog/components/NominationGroupEvaluationForm.tsx

```diff
@@ -0,0 +1,195 @@
+    ...NominationGroupEvaluationDialogLateralMovement
+    ...NominationGroupEvaluationDialogDevelopmentPrograms
+    id
+
+    nominee {
+      firstName
+    }
+
+    advancementNominationCount
+    lateralMovementNominationCount
+    developmentProgramsNominationCount
+
+    # for form initialState
+    advancementDecision {
+      value
+    }
+    advancementReferenceConfirmed
+    advancementNotes
+    lateralMovementDecision {
+      value
+    }
+    lateralMovementNotes
+    developmentProgramsDecision {
+      value
+    }
+    developmentProgramsNotes
+  }
+`);
+
+const NominationGroupEvaluationFormOptions_Fragment = graphql(/* GraphQL */ `
+  fragment NominationGroupEvaluationFormOptions on Query {
+    ...NominationGroupEvaluationDialogLateralMovementOptions
+  }
+`);
+
+interface NominationGroupEvaluationFormProps {
+  onSubmit: SubmitHandler<FormValues>;
+  talentNominationGroupQuery: FragmentType<
+    typeof NominationGroupEvaluationForm_Fragment
+  >;
+  talentNominationGroupOptionsQuery: FragmentType<
+    typeof NominationGroupEvaluationFormOptions_Fragment
+  >;
+}
+
+const NominationGroupEvaluationForm = ({
+  onSubmit,
+  talentNominationGroupQuery,
+  talentNominationGroupOptionsQuery,
+}: NominationGroupEvaluationFormProps) => {
+  const intl = useIntl();
+
+  const talentNominationGroup = getFragment(
+    NominationGroupEvaluationForm_Fragment,
+    talentNominationGroupQuery,
+  );
+  const talentNominationGroupOptions = getFragment(
+    NominationGroupEvaluationFormOptions_Fragment,
+    talentNominationGroupOptionsQuery,
+  );
+
+  const methods = useForm<FormValues>({
+    defaultValues: convertQueryDataToFormData(talentNominationGroup),
+  });
+  const { handleSubmit } = methods;
+
+  const isNominatedForAdvancement =
+    talentNominationGroup?.advancementNominationCount;
+  const isNominatedForLateralMovement =
+    talentNominationGroup?.lateralMovementNominationCount;
+  const isNominatedForDevelopmentPrograms =
+    talentNominationGroup?.developmentProgramsNominationCount;
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1.25)"
+        >
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x0.5)"
+          >
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "Ready to submit the evaluation of this nomination? Please complete the form for the selected nomination options.",
+                id: "eGlL1Q",
+                description:
+                  "Introduction for form to evaluate a nomination group",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage(
+                {
+                  defaultMessage: "{nomineeName} has been nominated for: ",
```

Comment:
It is kind of a annoying yo use but I think it is beneficial if we use the `commonMessages.dividingColon`

Reply:
Yeah, I can't imagine us ever needing to use that string without the colon so I'm not sure how useful that is, but if you prefer it.
[use dividing colon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13305/commits/13358372d4d03baa4ea9831fabb4b873c3bcd29e)